src/systems/systems/src/apu/APS3200.md

```diff
@@ -8,15 +8,15 @@ EGT effect (less air density causing higher EGT during start as it provides less
 
 ```
 N, EGT
-1, 0;
-2, 0;
-3, 0;
-4, 0;
-5, 0;
-6, 0;
-7, 0;
-8, 0;
-9, 0;
+1, -85;
```

Comment:
Negative 85 EGT? Is this some kind of corrected value?

Reply:
This data, when entered on http://polynomialregression.drque.net/online.php provides a function that will also work well when you for example start the APU at 40.000 feet with an OAT of -50. Before, the APU EGT would just skip to 0 immediately, which wasn't right.

We don't know (yet) what effect OAT and altitude have on the APU's EGT. Once we know their effect, we can make this formula more relative to those outside factors. Something for the future! 😀

Reply:
I see! Sounds good.